test/source/util/index.ts

```diff
@@ -52,34 +52,45 @@ export type TestKeyInfoWithFilepath = TestKeyInfo & { filePath?: string, expired
 interface TestSecretsInterface {
   ci_admin_token: string;
   auth: { google: { email: string, password?: string, secret_2fa?: string }[], };
-  keys: TestKeyInfo[];
 }
 
 export class Config {
 
   public static extensionId = '';
 
   private static _secrets: TestSecretsInterface;
+  private static _keys: TestKeyInfo[];
 
   public static secrets = (): TestSecretsInterface => {
     if (!Config._secrets) {
       try {
         Config._secrets = JSON.parse(fs.readFileSync('test/test-secrets.json', 'utf8'));
-        Config._secrets.keys = testKeyConstants.keys;
       } catch (e) {
         console.error(`skipping loading test secrets because ${e}`);
-        Config._secrets = { auth: { google: [] }, keys: [] } as any as TestSecretsInterface;
+        Config._secrets = { auth: { google: [] } } as any as TestSecretsInterface;
       }
     }
     return Config._secrets;
   };
 
+  public static keys = (): TestKeyInfo[] => {
+    if (!Config._keys) {
+      try {
+        Config._keys = JSON.parse(fs.readFileSync('test/test-keys.json', 'utf8'));
+      } catch (e) {
+        console.error(`skipping loading test keys because ${e}`);
+        Config._keys = [];
+      }
```

Comment:
No need to handle it graciously. Better let it throw upwards, so that the problem becomes more obvious if it happens.

Reply:
It will crash the test which should make it clear to the dev what happened.